src/docs/devices/Homemaster-MiniPLC/index.md

```diff
@@ -58,13 +59,34 @@ Ethernet Support – Optional LAN8720 PHY via RMII interface for reliable wired
 
 ## Programming
 
-The MiniPLC comes with ESPHome pre-installed and can be programmed via:
+The MicroPLC comes with ESPHome pre-installed and can be confgured via:
 
-Wi-Fi: Use the ESPHome Dashboard to upload the configuration.
+### Improve
 
-USB Type-C: Use the ESPHome Dashboard to upload the configuration.
+Wi-Fi Configuration with Improv
 
-Integrated Webserver: Access the MiniPLC's IP address, upload the configuration via the built-in web UI.
+1. Power on your HomeMaster MinioPLC.
+2. Go to 👉 improv-wifi.com.
+3. Connect via USB (Serial) or Bluetooth LE.
+4. Enter your Wi-Fi SSID and password, then press Connect.
+5. The device joins your Wi-Fi and is now ready.
+
+You can then access it via its local address (e.g., http://homemaster-microplc.local) or directly in Home Assistant.
```

Comment:
With `name_add_mac_suffix: true` the address will have the mac suffix. With the `web_server` in the config, `improv` will actually give a link to the user to browse there using the direct IP address.